app/auth/views.py

```diff
@@ -22,14 +22,15 @@
     ChangePasswordForm,
     ChangeUserDataForm
 )
-from .modules import check_password_requirements, get_supervisors_for_division, create_user, get_changelog_by_user_id, update_user_information
+from .modules import check_password_requirements, get_supervisors_for_division, create_user, get_changelog_by_user_id, update_user_information, TimePassed
 from .. import db
 from ..decorators import admin_required
 from ..email_notification import send_email
 from ..models import User, Role
 from ..utils import InvalidResetToken
 
 
+still_locked=TimePassed() #Just a timer
```

Comment:
We shouldn't have globals in the application. Let's discuss what hte actual goal is for this functionality.